docs/dev_guides/changing_splink/testing.md

```diff
@@ -30,6 +27,11 @@ or alternatively
 pytest tests/
 ```
 
+!!! information Note
+
+    If you do not have an AWS account, or have not set up Splink to use it, you will need to add
```

Comment:
It might be better to tweak the logic on this function. I've added a decorator in our [linkage repo](https://github.com/moj-analytical-services/data_linking/blob/main/tests/communicate_with_s3.py#L16) that detects whether you're logged in and able to access s3 and skips if not and seems to be more stable.

When I get to this step I'll briefly check it and create an issue if it seems to be worth the effort.

Reply:
The linkage repo is not public; I will apply Robin's suggested change for now and see if it works for me.

Reply:
It does 🎉 -- I'll leave further improvements to a future PR.